### PR TITLE
Override template's "hasStaticData" defaults 

### PR DESCRIPTION
**Pull request by rgwozdz**
Override server and layer template's hasStaticData property if provider's metadata has a similarly named value of type boolean.

### CODE REVIEW

File: test/info.js [hunk 1]
```diff
@@ -292,6 +292,19 @@ describe('Info operations', () => {
...
+      const input = {
+        metadata: {
+          hasStaticData: false,
+          geometryType: 'Polygon',
+          extent: [[11, 12], [13, 14]],
+          fields: [{ name: 'test', type: 'integer' }]
+        }
+      }
+      const layer = FeatureServer.layerInfo(input, {})
+      layer.hasStaticData.should.equal(false)
```
Comment (dmfenton) [inline test/info.js]: has static data should default to false

Comment (rgwozdz) [reply test/info.js]: Changed default to `false`, made test adjustments, and noted in changelog in 1f2cb24636bbb1c1bec0e9fc207cf8ff6fe352fe.

Comment (dmfenton) [reply test/info.js]: 👍

Comment (dmfenton) [reply test/info.js]: since this test is overriding, then it should be testing the `true` condition

**Comment by rgwozdz** [comment]
@dmfenton - ok to merge this now?

**Pull request merged**